content/intro-to-storybook/react/en/simple-component.md

```diff
@@ -117,23 +142,31 @@ Start by changing your Storybook configuration file (`.storybook/main.js`) to th
 module.exports = {
   stories: ['../src/components/**/*.stories.js'],
   addons: [
-    '@storybook/preset-create-react-app',
-    '@storybook/addon-actions',
     '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/preset-create-react-app',
   ],
 };
 ```
 
-After completing the change above, inside the `.storybook` folder, add a new file called `preview.js` with the following:
+After completing the change above, inside the `.storybook` folder, change the `preview.js` to the following:
 
 ```javascript
 // .storybook/preview.js
 
 import '../src/index.css';
+
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
Oh! If we did this we don't need to add the actions args.

Reply:
Yeah, perhaps we need some text in that spot to explain what's happening, WDYT?

Reply:
Added with the latest commit. Didn't want to go into it too much, gave a description of what are they and how they will be used. Also left a link pointing to the official documentation, should the reader wants to learn more about it.

Let me know if you're ok with it.